disnake/threads.py

```diff
@@ -231,7 +235,7 @@ def _update(self, data):
             pass
 
     @property
-    def type(self) -> ChannelType:
+    def type(self) -> ThreadType:
         """:class:`ChannelType`: The channel's Discord type."""
```

Comment:
i might want to update this docstring too.

Reply:
https://github.com/DisnakeDev/disnake/pull/543/commits/2d90aec43a03d7eb53fe16112815903425b2e895